### PR TITLE
My groups cards changes

### PR DESCRIPTION
**Pull request by lk316**
# Description

Profile groups style updates

## Issue ticket number and link

https://trello.com/c/m2mCHmfS/189-my-groups-visual-differences
https://trello.com/c/iy0xJIGD/195-use-default-placeholder-when-image-is-missing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/styles/components/content-cards.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.group-img-container {
```
Comment (ogabor82) [inline src/styles/components/content-cards.css:1]: Please use Tailwind css classes instead of custom css if it possible.

**Comment by lk316** [comment]
Made the requested changes, thanks @ogabor82

File: public/views/partials/organisms/content-card.liquid [hunk 2]
```diff
@@ -62,15 +62,20 @@ metadata:
...
       {% endif %}
         <picture>
-          {% if avifSrcSet != blank %}
-          <source type="image/avif" srcset="{{avifSrcSet}}">
+          {% if avifSrcSet != blank and imgSrc != blank %}
+            <source type="image/avif" srcset="{{avifSrcSet}}">
+            {% assign src = imgSrc %}
+            {% assign srcset = imgSrcSet %}
+          {% else %}
+            {% assign src = 'images/PCBS_-_Placeholder_image.png' | asset_url %}
```
Comment (nemvalid) [inline public/views/partials/organisms/content-card.liquid:75]: this creates a dependency between the components module and the pcbs app. this means that the module will only work for PCBS.
it should be the other way around: it's okay for an app to depend on a module, but it's never okay for a module to depend on the app.

**Review by nemvalid — CHANGES_REQUESTED** [review]

File: public/views/partials/organisms/content-card.liquid (hunk 2 above)
Comment (Slashek) [reply public/views/partials/organisms/content-card.liquid]: what would be the best way to solve this? having a constant as a configuration with the name of the placeholder? creating an `atom` for placeholder?

**Pull request merged**